kura/org.eclipse.kura.driver.opcua.provider/src/main/java/org/eclipse/kura/internal/driver/opcua/OpcUaDriver.java

```diff
@@ -225,7 +171,13 @@ public synchronized void connect() throws ConnectionException {
                 endPointBuilder.append("/").append(serverName);
             }
 
-            final EndpointDescription[] endpoints = UaTcpStackClient.getEndpoints(endPointBuilder.toString()).get();
+            String endpointString = endPointBuilder.toString();
+
+            logger.debug("Connecting to endpoint: {}", endpointString);
+
+            logger.debug("Fetching endpoint descriptions");
```

Comment:
My bad. I didn't notice it's a DEBUG log. Yes, you are right, it should not be localized.